backend/data/stove.json

```diff
@@ -21,10 +40,29 @@
     "brandName": "General Electric",
     "modelNumber": "PB965YPFS",
     "sourceUrl": "https://www.geappliances.com/appliance/GE-Profile-30-Smart-Free-Standing-Electric-Double-Oven-Convection-Range-with-No-Preheat-Air-Fry-PB965YPFS",
-    "weightInKg": 73.0284,
-    "widthInCm": 75.8825,
-    "heightInCm": 120.65,
-    "lengthInCm": 71.755,
+    "weights": [
+      {
+        "description": "weight",
+        "value": 73.0284,
+        "unit": "kg"
+      }
+    ],
```

Comment:
I've thought a bit more about it, and now I'm thinking we really want that will let us distinguish between what was originally collected and what was derived/normalized. We want to store both for auditing purposes.

So that could look one of two ways:

Way 1: stored in collected/derived keys at the top-level:
```
weight: {
    collected: [{
        value: 73,
        unit: "kg"
    }],
    finalized: [{
        value: 160,
        unit: "lb"
    },
    {
        value: 73,
        unit: "kg"
    }]
}
```

Both of the above are arrays since there could be more than one unit that we have measurements for. 

Alternatively, it could be:
```
weight: [
    {
         value: "kg",
         unit: 73,
         type: "collected"
    },
    {
         value: "kg",
         unit: 73,
         type: "collected"
    },
    {
         value: 160,
         unit: "lb",
         type: "finalized"
     }
]
```



The backend processing needs to run over the collecting units and generate the finalized units.

Then the frontend passes in a setting for whether they want the data in metric or imperial, and the backend needs to retrieve the appropriate finalized unit based on that.

We do have to agree that derived/finalized should be a *specific* metric or imperial measurement (e.g. if it's centimeters, and if the PDF is expressed in millimeters, we still need to  divide by 10 to get a derived/finalized unit in centimeters).

Reply:
For now, I think it will be easier to implement the second approach. If necessary, we can always make the change and convert the data back to the first approach.